src/hakunapi-jsonfg/src/main/java/fi/nls/hakunapi/geojson/hakuna/JSONFGFeatureCollectionWriter.java

```diff
@@ -0,0 +1,233 @@
+package fi.nls.hakunapi.geojson.hakuna;
+
+// TODO: forced to share package fi.nls.hakunapi.geojson.hakuna due to json.writeStringUnsafe scope
```

Comment:
Let's just change that to `protected`, no real point in hiding it as the extending class has raw access to the actual `OutputStream` and can mess things up if it wants to.

Reply:
changed to protected 
moved jsonfg to fi.nls.hakunapi.jsonfg